app/tools/android/make_apk.py

```diff
@@ -320,10 +320,6 @@ def CopyCompressedLibrary(native_path, library_path, raw_path, arch):
 
 
 def CopyNativeLibrary(native_path, library_path, raw_path, arch):
-  dummy_library = os.path.join(native_path, DUMMY_LIBRARY);
-  if os.path.isfile(dummy_library):
-    os.remove(dummy_library)
```

Comment:
By default lzma is enabled. if user decides to disable lzma, the function "CopyNativeLibrary()" is called, and it mistakenly delete the "libxwalkdummy.so" by running "os.remove(dummy_library)". That's why I deleted those three lines.
After modification, it will be same as it is in master. It is mistakenly introduced when implementing "--disable-lzma" which flag only exists in Lite.